.cirrus.yml

```diff
@@ -109,9 +111,29 @@ test_task:
         - gating
 
     script:
-        - "$SCRIPT_BASE/latest_podman.sh"
-        - "$SCRIPT_BASE/test.sh"
+        - ${SCRIPT_BASE}/enable_ssh.sh
+        - ${SCRIPT_BASE}/enable_podman.sh
+        - ${SCRIPT_BASE}/test.sh
+
+latest_task:
+    name: "Test Podman main on $FEDORA_NAME"
+    alias: latest
+    allow_failures: true
+
+    # see bors.toml
+    skip: $CIRRUS_BRANCH =~ ".*\.tmp"
```

Comment:
In fact...I think it was bors that required this `skip`, could maybe nuke all of them as well.